### PR TITLE
DTSPO-1455 temporary additional route for sds prod

### PR DESCRIPTION
**Pull request by rahman124**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: config/terraform/ss/vars/01-network/prod.tfvars [hunk 1]
```diff
@@ -50,5 +50,11 @@ additional_routes = [
     address_prefix         = "10.24.254.130/32"
     next_hop_type          = "VirtualAppliance"
     next_hop_in_ip_address = "10.11.8.36"
+  },
+  {
+    name                   = "sdsprod_temp"
```
Comment (timja) [inline config/terraform/ss/vars/01-network/prod.tfvars:55]: why is this needed? the next hob is already the hub? is there another route conflicting?

Comment (rahman124) [reply config/terraform/ss/vars/01-network/prod.tfvars:55]: @timja indeed the next hop is the hub anyways, however considering this is also peered to the VHUB , i suspect this is causing the issue for the data source to come through for prod.

i'll close this pr though and manually add the specific route temporarily to confirm my suspicions
